ProcessLib/ComponentTransport/ComponentTransportFEM.h

```diff
@@ -98,19 +111,19 @@ class LocalAssemblerData : public ComponentTransportLocalAssemblerInterface
         bool is_axially_symmetric,
         unsigned const integration_order,
         ComponentTransportProcessData const& process_data,
-        std::vector<std::reference_wrapper<ProcessVariable>> const&
-            per_process_variables)
+        std::vector<std::vector<std::reference_wrapper<ProcessVariable>>> const&
+            process_variables)
```

Comment:
If there aren't any important reasons I would like to keep the previous version.

Reply:
Please take a look at line 539:
```
auto const& component = phase.component(
            _process_variables[transport_process_id][0].get().getName());
```
You may understand my motivation.

Reply:
Okay, I see you need the process variable for the hydraulic process and process variables for the for the chemical components. Below, you can to write:
`auto const& hydraulic_process_variable = _process_variables[hydraulic_process_id][0];`
IMHO this would make the code more readable.

Reply:
Fixed. `_transport_process_variables` introduced in local assembler.